src/component/Install.js

```diff
@@ -0,0 +1,118 @@
+import React, {useState} from 'react';
+import {Button, Datepicker, Select} from '@datorama/app-components';
+import sdk from '@datorama/sdk';
+import styled from "styled-components";
+import {useDispatch, useSelector} from 'react-redux';
+import {changeDataStream, changeDateRange} from '../redux/actions/todo.actions';
+import moment from "moment";
+
+const Install = () => {
+    const dispatch = useDispatch();
+    const installConfig = useSelector(state => state.todo);
+
+    const handleClick = () => {
+        console.log(installConfig);
+        sdk.app.install({config: installConfig});
+    };
+
+    const [selected, setSelected] = useState([]);
+    const handleDateRangeClick = (event) => {
+        dispatch(changeDateRange(event));
+    };
+
+    const handleDataStreamClick = event => {
+        setSelected(event);
```

Comment:
i see that you store the selected in the component state and also in the redux state. it's a duplication - instead you can read the data from redux and delete the selected / setSelected.

const selectedDataStream = installConfig.datastream.  // for example